core/src/main/java/info/nightscout/androidaps/utils/extensions/UserEntryExt.kt

```diff
@@ -1,224 +1,230 @@
-        UserEntry.Action.TT_REMOVED                 -> R.color.tempTargetConfirmation
-        UserEntry.Action.NS_PAUSED                  -> R.color.defaulttext
-        UserEntry.Action.NS_QUEUE_CLEARED           -> R.color.defaulttext
-        UserEntry.Action.NS_SETTINGS_COPIED         -> R.color.defaulttext
-        UserEntry.Action.ERROR_DIALOG_OK            -> R.color.defaulttext
-        UserEntry.Action.ERROR_DIALOG_MUTE          -> R.color.defaulttext
-        UserEntry.Action.ERROR_DIALOG_MUTE_5MIN     -> R.color.defaulttext
-        UserEntry.Action.OBJECTIVE_STARTED          -> R.color.defaulttext
-        UserEntry.Action.OBJECTIVE_UNSTARTED        -> R.color.defaulttext
-        UserEntry.Action.OBJECTIVES_SKIPPED         -> R.color.defaulttext
-        UserEntry.Action.STAT_RESET                 -> R.color.defaulttext
-        UserEntry.Action.DELETE_LOGS                -> R.color.defaulttext
-        UserEntry.Action.DELETE_FUTURE_TREATMENTS   -> R.color.defaulttext
-        UserEntry.Action.EXPORT_SETTINGS            -> R.color.defaulttext
-        UserEntry.Action.IMPORT_SETTINGS            -> R.color.defaulttext
-        UserEntry.Action.RESET_DATABASES            -> R.color.defaulttext
-        UserEntry.Action.EXPORT_DATABASES           -> R.color.defaulttext
-        UserEntry.Action.IMPORT_DATABASES           -> R.color.defaulttext
-        UserEntry.Action.OTP_EXPORT                 -> R.color.defaulttext
-        UserEntry.Action.OTP_RESET                  -> R.color.defaulttext
-        UserEntry.Action.SMS_BASAL                  -> R.color.basal
-        UserEntry.Action.SMS_BOLUS                  -> R.color.iob
-        UserEntry.Action.SMS_CAL                    -> R.color.calibration
-        UserEntry.Action.SMS_CARBS                  -> R.color.carbs
-        UserEntry.Action.SMS_EXTENDED_BOLUS         -> R.color.extendedBolus
-        UserEntry.Action.SMS_LOOP_DISABLED          -> R.color.loopDisabled
-        UserEntry.Action.SMS_LOOP_ENABLED           -> R.color.loopClosed
-        UserEntry.Action.SMS_LOOP_RESUME            -> R.color.loopClosed
-        UserEntry.Action.SMS_LOOP_SUSPEND           -> R.color.loopSuspended
-        UserEntry.Action.SMS_PROFILE                -> R.color.white
-        UserEntry.Action.SMS_PUMP_CONNECT           -> R.color.loopDisconnected
-        UserEntry.Action.SMS_PUMP_DISCONNECT        -> R.color.loopDisconnected
-        UserEntry.Action.SMS_SMS                    -> R.color.defaulttext
-        UserEntry.Action.SMS_TT                     -> R.color.tempTargetConfirmation
-        UserEntry.Action.TT_DELETED_FROM_NS         -> R.color.tempTargetConfirmation
-        UserEntry.Action.TT_FROM_NS                 -> R.color.tempTargetConfirmation
-        UserEntry.Action.TT_CANCELED_FROM_NS        -> R.color.tempTargetConfirmation
-        else                                        -> R.color.defaulttext
+        Action.EXTENDED_CARBS             -> R.color.carbs
```

Comment:
Shouldn't it be based on `ColorGroup`?

Reply:
With ColorGroup, Action.colorId is not used anymore. I can remove it.